Mobile sticky menu

iFlameing opened the pull request:
<img width="398" height="842" alt="clientimage" src="https://github.com/user-attachments/assets/bca1c2e0-012e-4932-84ed-407eb49b377f" />

sneridagh commented:
@iFlameing the links are not working on my side (tested with mobile chrome simulator).

iFlameing commented:
> @iFlameing the links are not working on my side (tested with mobile chrome simulator). The upgrade step is not needed also, you can safely remove it.

Fixed links and removed upgrade steps.

iFlameing commented:
@sneridagh @davisagli can you please review this pr.

sneridagh reviewed (approved):
LGTM, @danalvrz merging.
